exporter/opensearchexporter/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenSearch Exporter
+
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [devel]   |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
+
+This exporter supports sending OpenTelemetry signals as documents to [OpenSearch](https://www.opensearch.org).
+
+The documents are sent using [observability catalog](https://github.com/opensearch-project/opensearch-catalog/tree/main/schema/observability) schema.
+
+## Configuration options
+
+- `endpoints`: List of OpenSearch URLs. If endpoints is missing, the
+  OPENSEARCH_URL environment variable will be used.
+- `num_workers` (optional): Number of workers publishing bulk requests concurrently.
+- `dataset` (default=`default`) a user-provided label. It is used to construct the name of the destination index or data stream.
+- `namespace` (default=`namespace`)a user-provided label. It is used to construct the name of the destination index or data stream.
+- `flush`: Event bulk buffer flush settings
+    - `bytes` (default=5242880): Write buffer flush limit.
+    - `interval` (default=30s): Write buffer time limit.
+- `retry`: Event retry settings
+    - `enabled` (default=true): Enable/Disable event retry on error. Retry
+      support is enabled by default.
+    - `max_requests` (default=3): Number of HTTP request retries.
+    - `initial_interval` (default=100ms): Initial waiting time if a HTTP request failed.
+    - `max_interval` (default=1m): Max waiting time if a HTTP request failed.
+
+### HTTP settings
```

Comment:
This should use the [`confighttp.HTTPClientSettings` struct](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/confighttp.go#L31) to provide consistency of configuration and functionality across exporters.  Using `confighttp` can cover for the HTTP, [TLS](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/configtls.go#L65), and AuthN settings included here.

Reply:
Yes, I think this is feasible.